### ISSUE TITLE
calculation of FCC matrix fails because of large number of chains

### ISSUE THREAD

**Issue by amjjbonvin** [haddocking/fcc#7]
I am trying to cluster a complex consisting of 27 chains. `calc_fcc_matrix.py` fails with the following error message:

```
rackham.abonvin.Target170-scoring/rescored> python /home/abonvin/haddock_git/fcc/scripts/calc_fcc_matrix.py -f file.contacts -o rmsds/fcc.out
+ BEGIN: Tue Aug  4 16:58:59 2020
+ Parsing 951 contact files
Traceback (most recent call last):
  File "/home/abonvin/haddock_git/fcc/scripts/calc_fcc_matrix.py", line 129, in <module>
    c = parse_contact_file(args, exclude_chains)
  File "/home/abonvin/haddock_git/fcc/scripts/calc_fcc_matrix.py", line 20, in parse_contact_file
    contacts = [ set([ int(l) for l in open(f)]) for f in f_list if f.strip()]
ValueError: invalid literal for int() with base 10: '10001-1110031-10\n'
```

I did a fresh pull request before running it. Something is wrong with the contacts possibly.

**Comment by JoaoRodrigues** [comment]
Thanks for reporting @amjjbonvin , will have a look!

**Comment by JoaoRodrigues** [comment]
This had been solved in #5
